pkg/api/defaults-apiserver.go

```diff
@@ -164,6 +164,13 @@ func (cs *ContainerService) setAPIServerConfig() {
 	if common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.20.0-alpha.1") && o.KubernetesConfig.APIServerConfig["--service-account-issuer"] == "kubernetes.default.svc" {
 		o.KubernetesConfig.APIServerConfig["--service-account-issuer"] = "https://kubernetes.default.svc.cluster.local"
 	}
+
+	outdatedFeatureGates := []string{}
```

Comment:
not sure `outdated` is the term that reflects the issue, `invalid` may be slightly better?

Reply:
Agreed, changed to invalid